src/components/HexagonTile.tsx

```diff
@@ -0,0 +1,34 @@
+import { Text } from "@react-three/drei";
+import GameElement from "./GameElement";
+
+const HexagonTile = ({
+  position,
+  color,
+  name = "",
+}: {
+  position: [number, number, number];
+  color: string;
+  name?: string;
+}) => {
+  return (
+    <GameElement
+      position={position}
+      rotation={[-1.57, 0, 0]}
+      height={6}
+      width={10}
+    >
+      <cylinderGeometry args={[5, 5, 0.1, 6, 1]} />
```

Comment:
Haha, cool I changed it to triangles now, because I realised that 6 pentagons do not fit into eachother perfectly like hexagons do, so now they are all triangles pointing to the middle.